iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/StaticVarCompensatorImpl.java

```diff
@@ -95,44 +95,64 @@ public StaticVarCompensatorImpl setBmax(double bMax) {
         return this;
     }
 
+    @Override
+    public double getVoltageSetpoint() {
+        return voltageSetpoint.get(getNetwork().getVariantIndex());
+    }
+
     @Override
     public double getVoltageSetPoint() {
-        return voltageSetPoint.get(getNetwork().getVariantIndex());
+        return getVoltageSetpoint();
```

Comment:
If you add a default implementation in the interface, you wil not have to implement the deprecated method here